datafusion_ray/context.py

```diff
@@ -153,19 +141,24 @@ def sql(self, sql: str) -> pa.RecordBatch:
             return []
 
         df = self.df_ctx.sql(sql)
-        execution_plan = df.execution_plan()
+        return self.plan(df.execution_plan())
+
+    def plan(self, execution_plan: Any) -> pa.RecordBatch:
 
         graph = self.ctx.plan(execution_plan)
         final_stage_id = graph.get_final_query_stage().id()
-        partitions = schedule_execution(graph, final_stage_id, True)
+        # serialize the query stages and store in Ray object store
+        query_stages = [
```

Comment:
I am a little puzzled, are these the query_stages or the serialized execution plans?

Reply:
these are the serialized execution plans for each query stage:

```python
graph.get_query_stage(i).get_execution_plan_bytes()
```